src/engraving/dom/engravingitem.h

```diff
@@ -403,7 +403,7 @@ class EngravingItem : public EngravingObject
 
     bool isPrintable() const;
     bool isPlayable() const;
-    double point(const Spatium sp) const { return sp.val() * spatium(); }
+    virtual double point(const Spatium sp) const { return sp.val() * spatium(); }
```

Comment:
I like what this method does, but I find the name a bit misleading. "Point" for me is immediately associated to an (x, y) pair of data. Can we think of something a bit clearer? Perhaps something like `double absoluteLenFromSpatium(const Spatium)`, which could have in future a corresponding opposite `absoluteLenToSpatium`. @cbjeukendrup what do you think?

Also, pure pedantry: the const keyword isn't really necessary when the parameter is passed by copy (as opposed to passing by reference, where const makes a big difference).

Reply:
Or even just `absoluteFromSpatium`k, as it's not always necessarily a "length".